source/content/wordpress-launch-check.md

```diff
@@ -18,56 +20,67 @@ This mechanism does not actually perform requests on your site, and in doing so
 In short, you get a fast, repeatable report that can help detect common problems and provide introspection into your site.
 
 ## How Does it Work?
-WP Launch Check is an extension for WP-CLI designed for Pantheon customers. While designed initially for the Pantheon Dashboard it is intended to be fully usable outside of Pantheon.
 
-To use WP Launch Check from the command line, run: `wp launchcheck <subcommand>`.
+WP Launch Check is a site audit extension for WP-CLI designed for Pantheon customers. While designed initially for the Pantheon Dashboard it is intended to be fully usable outside of Pantheon.
 
-For more information about WP-CLI, visit their [GitHub page](https://github.com/wp-cli/wp-cli). For more information on WordPress Launch Check, go to the [GitHub repo](https://github.com/pantheon-systems/wp_launch_check/).
+## Run Launch Check Manually
 
+To manually perform a site audit using WP Launch Check from the command line, run: `wp launchcheck <subcommand>`.
```

Comment:
I didn't adjust this in my edits because I wasn't sure of two things:

1. Should this be changed to a code block to match the way it's presented in the Drupal doc?
1. Should it be wrapped in Terminus so it can be executed on the platform?